src/plugins/intel_gpu/src/graph/gemm.cpp

```diff
@@ -132,6 +132,9 @@ std::vector<layout> gemm_inst::calc_output_layouts(gemm_node const& node, const
                                                                           prim->output_transpose_order);
```

Comment:
(random spot)
From your diagram, input 0 is 2d and input 1 is 5d. How is the operation defined in such case? Do we have some spec for such situation?

Reply:
Actually we don't have gemm op spec. Input1 5d is [batch,featue,z,K,N].